psl-core/src/main/java/org/linqs/psl/reasoner/term/MemoryVariableTermStore.java

```diff
@@ -93,24 +94,49 @@ public float getVariableValue(int index) {
     }
 
     @Override
-    public void syncAtoms() {
+    public double syncAtoms() {
+        double movement = 0.0;
+
         for (int i = 0; i < variables.size(); i++) {
+            movement += Math.pow(variableAtoms[i].getValue() - variableValues[i], 2);
             variableAtoms[i].setValue(variableValues[i]);
         }
+
+        return movement;
```

Comment:
Should we root and average the movement?
Essentially: SE, MSE, or RMSE?
My inclination is towards RMSE, but I'm not super strong in any direction here.